url-plugins/checkstyle.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wonder if we could have one copy of this at the root level, instead of in this/each directory.